plugins/browser-plugin-media-tracking/README.md

```diff
@@ -0,0 +1,87 @@
+# Snowplow Media Tracking
+
+[![License][license-image]](LICENSE)
+
+Browser Plugin to be used with `@snowplow/browser-tracker`.
+
+Adds HTML5 Video and Audio tracking events to your Snowplow tracking.
+
+## Maintainer quick start
+
+Part of the Snowplow JavaScript Tracker monorepo.  
+Build with [Node.js](https://nodejs.org/en/) (12 LTS or 14 LTS) and [Rush](https://rushjs.io/).
+
+### Setup repository
+
+```bash
+npm install -g @microsoft/rush 
+git clone https://github.com/snowplow/snowplow-javascript-tracker.git
+rush update
+```
+
+### Package Installation
+
+With npm:
+
+```bash
+npm install @snowplow/browser-plugin-media-tracking
+```
+
+## Usage
+
+Initialize your tracker with the MediaTrackingPlugin:
+
+```js
+import { newTracker } from '@snowplow/browser-tracker';
+import { MediaTrackingPlugin } from 'snowplow-browser-media-tracker';
+
+newTracker('sp2', '{{collector}}', { plugins: [ MediaTrackingPlugin() ] }); // Also stores reference at module level
+```
+
+Then, use the `enableMediaTracking` function described below to produce events from your HTML5 Video/Audio element(s).
+
+```js
+enableMediaTracking({ id, options?: { label?, captureEvents?, percentBoundries? } })
+```
+
+| Parameter                      | Type       | Default             | Description                                               | Required |
+| ------------------------------ | ---------- | ------------------- | --------------------------------------------------------- | -------- |
+| `id`                           | `string`   | -                   | The HTML id attribute of the media element                | Yes      |
+| `label`                        | `string`   | -                   | An identifiable custom label sent with the event          | No       |
+| `captureEvents`                | `string[]` | `['DefaultEvents']` | The name(s) of the events to capture                      | No       |
```

Comment:
Would be good to list in the readme the available options for this "DefaultEvents" and "AllEvents" and then maybe list out all the individual events at the bottom of the readme perhaps?